default.lcf

```diff
@@ -92,3 +92,7 @@ log4j.category.org.eclipse.jetty.server.handler=WARN
 # log4j.category.jmri.jmrix.can=DEBUG
 # log4j.category.jmri.jmrix.nce.NceTrafficController=WARN
 # log4j.category.org.eclipse.jetty=INFO
+log4j.category.jmri.jmrix.dccpp=DEBUG
```

Comment:
This needs to be silenced. Generally we want minimal logging in the default configuration.